### PR TITLE
NEWTONX interface - writing CP2K output

### PR DESCRIPTION
**Pull request by annahehn**
This is a small change to write output for the NEWTONX interface.

### CODE REVIEW

File: src/qs_tddfpt2_restart.F [hunk 1]
```diff
@@ -287,5 +291,127 @@ FUNCTION tddfpt_read_restart(evects, evals, gs_mos, logger, tddfpt_section, tddf
...
+               ! how eigenvectors are distributed across computational cores. The phase is important
+               ! because TDDFPT wave functions are used to compute a response electron density
+               ! \rho^{-} = 1/2 * [C_{0} * evect^T + evect * C_{0}^{-}], where C_{0} is the expansion
+               ! coefficients of the reference ground-state wave function. To make the restart file
+               ! transferable, TDDFPT wave functions are stored in assumption that all ground state
+               ! MOs have a positive phase.
+               CALL cp_fm_column_scale(evects(ispin, istate)%matrix, gs_mos(ispin)%phases_occ)
+
+               IF (ounit > 0) THEN
+                  WRITE (ounit, "(A)") "                    "
```
Comment (mkrack) [inline src/qs_tddfpt2_restart.F]: What is the point of these blanks?

Comment (annahehn) [reply src/qs_tddfpt2_restart.F]: None, sorry, I will remove them.

Comment (mkrack) [reply src/qs_tddfpt2_restart.F]: If you just want an empty line before a header use
```
WRITE (ounit, "(/,A)") "ES EIGENVECTORS SIZE"
```

Comment (annahehn) [reply src/qs_tddfpt2_restart.F]: ok, thanks!

Comment (annahehn) [reply src/qs_tddfpt2_restart.F]: I will recommit in a second

Comment (annahehn) [reply src/qs_tddfpt2_restart.F]: I adjusted now, thanks :)

**Pull request merged**